docker_tests/smt_server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.scc.suse.de/connect_sp2
+
+ADD files/smt_current.rpm /root/smt_current.rpm
+
+RUN zypper ref && zypper --non-interactive install --no-recommend \
+    timezone patch mariadb-client libopenssl-devel
+
+RUN zypper --non-interactive in --no-recommend /root/smt_current.rpm
```

Comment:
to prevent trouble with unknown GPG keys who signed this RPM you can call it with

    zypper --non-interactive --no-gpg-checks in --no-recommend /root/smt_current.rpm

Than gpg check results are ignored.